recipes/autoesda/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "autoesda" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 83bfa06deaa41a29979918261fa69a1f53c623012b25d8e2cf1bd98fb3e597b8
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
```

Comment:
The upstream package pins the versions of many the dependencies to a specific version. I just wanted to double check that having no version restrictions is known to be ok.

Reply:
This is fine